cltk/tokenize/indian_tokenizer.py

```diff
@@ -24,7 +24,7 @@ def indian_punctuation_tokenize_regex(input_str):
     :rtype: list
     """
     tok_str = indian_punctuation_pattern.sub(r' \1 ',input_str.replace('\t',' '))
-    return re.sub(r'[ ]+',u' ',tok_str).strip(' ').split(' ')
+    return re.sub(r'\s+',u' ',tok_str).strip(' ').split(' ')
```

Comment:
This change was made ,keeping the thought that if the sentence is "aaaa\naaaa" , in this case "aaaa" and "aaaa" are two different words but would have been treated a single word